backend/tools/db_viewer.py

```
@@ -0,0 +1,127 @@
+    for item in items:
+        # Format datetime strings to show just date
+        created = item[11][:10] if item[11] else 'N/A'
+        updated = item[12][:10] if item[12] else 'N/A'
+        
+        print(f"{item[0]:<3} {item[1] or 'N/A':<10} {item[2][:14] or 'N/A':<15} {item[3][:7] or 'N/A':<8} {item[4][:19] or 'N/A':<20} {item[5] or 1:<3} {item[6][:11] or 'N/A':<12} {item[7][:9] or 'N/A':<10} {item[8][:7] or 'N/A':<8} {item[9][:7] or 'N/A':<8} {item[10][:5] or 'N/A':<6} {created:<10} {updated:<10}")
+    
+    print(f"\nTotal items: {len(items)}")
+    
+    # Show detailed view of last 3 items
+    if items:
+        print(f"\nDETAILED VIEW (Last 3 items):")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)